src/main/java/se/diabol/jenkins/pipeline/domain/Component.java

```diff
@@ -31,19 +36,34 @@
     private final String firstJob;
     private final String firstJobUrl;
     private final boolean firstJobParameterized;
+    
+    private final int noOfPipelines;
+    
+    private int componentNumber = 0;
+    
+    private String pagingData = null;
+    
+    private boolean enablePaging = false;
 
     public Component(String name, String firstJob, String firstJobUrl, boolean firstJobParameterized,
-            List<Pipeline> pipelines) {
+            List<Pipeline> pipelines, int noOfPipelines, boolean enablePaging) {
         super(name);
         this.pipelines = ImmutableList.copyOf(pipelines);
         this.firstJob = firstJob;
         this.firstJobUrl = firstJobUrl;
         this.firstJobParameterized = firstJobParameterized;
+        this.noOfPipelines = noOfPipelines;
+        this.enablePaging = enablePaging;
     }
 
     @Exported
     public List<Pipeline> getPipelines() {
-        return pipelines;
+    	if(enablePaging) {
```

Comment:
Inconsistent formatting - missing a space between `if ` and `(`. In multiple places.